emoji/src/test/java/com/vanniktech/emoji/EmojiManagerTest.java

```diff
@@ -49,10 +47,61 @@
     assertThat(EmojiManager.getInstance().getCategories()).isNotEmpty();
   }
 
+  @Test
+  public void noProviderInstalled() {
```

Comment:
This test is currently failing because `EmojiManager` is a singleton and there is a `EmojiProvider` from a previous test installed.
This led me to the thought of adding a `EmojiManager#destroy` method which would remove the current `EmojiProvider` and clean up all internal data-structures.
This would allow apps to free the memory when they know that the `EmojiManager` is not needed anymore (The user exits the screen with the emoji input for example).

What do you think of this?